app/views/moderations/index.html.erb

```diff
@@ -31,12 +31,15 @@
   </main>
 <% else %>
   <div class="container" style="margin-top: 90px;">
-    <h1 style="text-align: center"><%= community_name %> Mods</h1>
+    <h1 class="pt-5" style="text-align: center"><%= community_name %> Mods</h1>
     <div class="body">
       <p>We periodically award some <%= community_name %> members with heightened privileges to help moderate the community.</p>
-      <p>Email <%= email_link %> if you'd like to be considered right away.</p>
+      <% if community_name == "DEV" %>
```

Comment:
As we've already had an issue about using `community_name` as a condition and since `community_name` on dev.to now is `DEV Community`, I think we should stick with `SiteConfig.dev_to?`

See the recent change here https://github.com/forem/forem/pull/12083

Reply:
Thank you Rhymes! 🙏🏾